src/fount.mjs

```
@@ -61,30 +62,64 @@ export async function unloadChar() {
 		chat_name: 'CI',
-		chat_id: 0,
 		char_id: charname,
 		username,
 		UserCharname: username,
+		UserUid,
 		Charname: Object.values(char.info || {})[0]?.name || charname,
+		CharUid,
 		locales: ['en-UK'],
-		chat_log: [],
+		time: new Date(),
+		chat_log,
```

Biome (2.5.6)

[error] This property is later overwritten by an object member with the same name.

(lint/suspicious/noDuplicateObjectKeys)